react/features/base/lib-jitsi-meet/functions.js

```diff
@@ -60,3 +60,45 @@ export function loadConfig(host: string, path: string = '/config.js') {
             throw err;
         });
 }
+
+/**
+ * Creates a JitsiLocalTrack for video from the given device id.
+ *
+ * @param {string} deviceId - The id of the target video source.
+ * @returns {Promise<JitsiLocalTrack>}
+ */
+export function createLocalVideoTrackPreview(deviceId) {
+    return createLocalTrack([ 'video' ], deviceId);
+}
+
+/**
+ * Creates a JitsiLocalTrack for audio from the given device id.
+ *
+ * @param {string} deviceId - The id of the target audio source.
+ * @returns {Promise<JitsiLocalTrack>}
+ */
+export function createLocalAudioTrackPreview(deviceId) {
+    return createLocalTrack([ 'audio' ], deviceId);
+}
+
+/**
+ * Creates a JitsiLocalTrack model from the given device id.
+ *
+ * @param {Array<string>} type - The types of tracks being created. Expected
+ * values are "video" and "audio".
+ * @param {string} deviceId - The id of the target media source.
+ * @returns {Promise<JitsiLocalTrack>}
+ * @private
+ */
+function createLocalTrack(type, deviceId) {
```

Comment:
I'll make a change to expose this function and have it expect a string for type.